recipes/libuv/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class libuvConan(ConanFile):
+    name = "libuv"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libuv.org"
+    description = "A multi-platform support library with a focus on asynchronous I/O"
+    topics = ("libuv", "asynchronous", "io", "networking", "multi-platform", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "cmake"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/*"
+    ]
+
+    @property
+    def _source_subfolder_name(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure()
+        return cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
```

Comment:
Done: 27448f9338c4f66e0594fe3dea4da6097d36edbd

Reply:
Since there were no problems with [msbuild build](https://c3i.jfrog.io/c3i/misc/logs/pr/1100/3/libuv/1.34.2/ff82a1e70ba8430648a79986385b20a3648f8c19/create_stdout.txt):

> -- Build files have been written to: C:/J/c3ipr/pr_1100_3_0_8/.conan/data/libuv/1.34.2/_/_/build/ff82a1e70ba8430648a79986385b20a3648f8c19
Microsoft (R) Build Engine version 15.9.21+g9802d43bc3 for .NET Framework

and I use CMake, that can work not only with Ninja, I don't think Ninja must present in `build_requirements`.